### ISSUE TITLE
Why using A regressor instead of a classifier for bounce detection ? 

### ISSUE THREAD

**Issue by valentin-fngr** [yastrebksv/TrackNet#6]
Hi, 

Basically title. 
Did you experiment with both types of algos and figured out regressor worked best ? 
Thanks

**Comment by yastrebksv** [comment]
Yep, I tried both regression and classification tasks. Regression worked better
